scripts/build.js

```diff
@@ -0,0 +1,64 @@
+const exec = require('execa');
```

Comment:
Rewrite this is typescript and use `ts-node` to execute it. Conform it to the lint & 2 spaces.